package.json

```
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "autoprefixer": "^10.4.14",
-    "postcss": "^8.4.23",
+    "postcss": "^8.4.31",
```

I think this is a better version:

```suggestion
    "autoprefixer": "10.4.14",
```